Base/res/html/misc/welcome.html

```diff
@@ -162,6 +162,7 @@ <h2>JavaScript/Wasm</h2>
             <li><h3>Wasm</h3></li>
             <li><a href="mandelbrot-wasm.html">WebAssembly Mandelbrot Rendering Demo</a></li>
             <li><a href="gol-wasm.html">WebAssembly Game Of Life Demo</a></li>
+            <li><a href="worker_parent.html">Workers</a></li>
```

Comment:
You didn't actually include the `worker_parent.html` file
(Also while we're here it should probably be at the bottom of the javascript section not in Canvas or WASM, just above `<li><h3>Canvas</h3></li>` seems fine.